Automattic/jetpack
# Repo Gardening: do not gather support refs on Pull Requests

## Proposed changes:

This PR includes 2 changes:

1. We should only gather issue references from issues. However, the event trigger we use (`issue_comment`) targets both PRs and issues. Let's bail early when we're on a Pull Request. See [the documentation here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment-on-issues-only-or-pull-requests-only)
2. When gathering p2 comment references (this was introduced in #33979), let's not change the capitalization of references.

### Other information:


## Jetpack product discussion

Internal reference: p1701430031933879-slack-C03N25JPCE4

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

This can only be tested in a fork. See these issues:

https://github.com/jeherve/jetpack/issues/126#issuecomment-1841123896
https://github.com/jeherve/jetpack/pull/127
